src/librustc_resolve/lib.rs

```diff
@@ -3125,6 +3125,31 @@ impl<'a> Resolver<'a> {
         })
     }
 
+    pub fn traits_in_scope(&mut self, module_id: DefId) -> Vec<TraitCandidate> {
```

Comment:
So, this should be equivalent to `get_traits_containing_item`, just without the "`_containing_item`" part?
Could you add a comment saying that? (And also saying that this function is for rustdoc.)

Reply:
Done, thanks for the review!